### DISCUSSION TITLE
Wanneer gaan we naar school?

### DISCUSSION THREAD

**Discussion by D0nnaz** [Sensinki/Blok-Tech-Team-4A#24] [General]
Welke dagen en hoe laat?

**Comment by rosa33783** [comment]
elke dag deze week kan ik naar school

**Comment by Sensinki** [comment]
Ik kan heledag op dinsdag en woensdag vanaf 13.00 uur tot 20.00 op school

**Comment by rosa33783** [comment]
we kunnen elke dag tot 17.00 of 18.00?
